### PR TITLE
Update vendor

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: app/config/routing.yml [hunk 1]
```diff
@@ -18,8 +18,5 @@ open_orchestra_redirection:
     resource: '.'
     type: orchestra_redirection
 
-open_orchestra_base:
```
Comment (itkg-ngilain) [inline app/config/routing.yml]: How do you generate the front url from the back if you remove this one ?

Comment (nicolasThal) [inline app/config/routing.yml]: The route given in the base bundle is already given by the ` open_orchestra_front_preview` entry.

Moreover the second entry is overriding the one in the base bundle, so there is no need to add the base-bundle routing

Comment (nicolasThal) [inline app/config/routing.yml]: And this installation is only for the front, so we do not generate  the preview url

Comment (itkg-ngilain) [inline app/config/routing.yml]: True, I thinked this was the back install when I wrote the comment ...

**Pull request merged**